tests/__init__.py

```
@@ -0,0 +1,31 @@
+        key: os.environ.get(key)
+        for key in [
+            "SMTP_USERNAME",
+            "SMTP_PASSWORD",
+            "ALLOWED_MAILFROM",
+            "REDIS_URL",
+            "DATABASE_URL",
+        ]
+    }
+
+    os.environ["SMTP_USERNAME"] = "test@example.com"
+    os.environ["SMTP_PASSWORD"] = "testpassword"
```

Ruff (0.14.10)

Possible hardcoded password assigned to: "SMTP_PASSWORD"

(S105)